### PR TITLE
Add fake-prison-register-api to pull from quay

### PR DESCRIPTION
**Pull request by NataliaOsk**

### CODE REVIEW

File: scripts/start-local.sh [hunk 1]
```diff
@@ -11,12 +11,16 @@ docker compose -f "${DOCKER_COMPOSE_FILE}" stop
 docker rm fake-manage-recalls-api || true
 docker build fake-manage-recalls-api
 
+docker rm fake-prison-register-api || true
+docker build fake-prison-register-api
```
Comment (dazoakley) [inline scripts/start-local.sh:15]: You don't need to do a `rm`/`build` here if you're pulling from a pre-built image.

Comment (NataliaOsk) [reply scripts/start-local.sh:15]: Got it, thanks! Will remove

File: docker-compose.yml [hunk 2]
```diff
@@ -35,12 +35,23 @@ services:
       test: [ "CMD", "curl", "-f", "http://localhost:8080/health/liveness" ]
     command: --verbose
 
+  fake-prison-register-api:
+    image: quay.io/hmpps/ppud-fake-prison-register-api:latest
+    networks:
+      - manage-recalls-ui
+    container_name: fake-prison-register-api-e2e
```
Comment (jonnywyatt) [inline docker-compose.yml:42]: this is used for integration tests as well, maybe don't need the suffix?

**Review by jonnywyatt — APPROVED** [review]

File: scripts/start-local.sh (hunk 1 above)
Comment (dazoakley) [reply scripts/start-local.sh:14]: Remove the `build` step too 😉

**Pull request closed without merging**